README.md

```diff
@@ -36,11 +36,10 @@ The message is published via amazon sns. The payload is defined below.
 }
 ```
 
-This service subscribes to two types of events:
-* prison-offender-events.prisoner.received
-* restricted-patients.patient.removed 
+This service subscribes to a `prison-offender-events.prisoner.received`, which indicates that a prisoner has been
+received into prison.  We check to see if the prisoner was a restricted patient and remove them if necessary.
 
-Restricted patients are removed automatically when a `prison-offender-events.prisoner.received` domain event has been received.
-
-When the service receives a `restricted-patients.patient.removed ` the event will logged and then ignored. This is for ease of testing.
```

Comment:
can't really see that it makes it much easier to test since it is additional code.  The event appears in offender events ui anyway so can be checked there to ensure that it is published.